### DISCUSSION TITLE
Saving/Exporting in non-GDS formats

### DISCUSSION THREAD

**Discussion by mpanipi** [HelgeGehring/gdshelpers#58] [Q&A]
EDIT: This issue can be solved by installing the fatamorgana package.

Hello!

I'm a new-minted grad student who is somewhat familiar with Python and fairly unfamiliar with gdsHelpers. I'm working on a nanophotonics project and am having some difficulty with the file sizes involved. I am looking at file types that are not GDS files to see if they are smaller, but I am having issues exporting the data as an .oasis file. 

Saving as an .oasis file seems pretty straightforward in the documentation: "The ending of the filename defines the format. Currently .gds, .oasis and .dxf are supported." Using cell.save('filename.oasis') doesn't throw an error, but it does yield a  gds file titled 'filename.oasis.gds'. I have attached an image of a segment of code that demonstrates this. This same code exports a .gds file just fine.

I also used the same piece of code to try and export a .dxf just to see what would happen and the result was an error, which is also shown below.

Does anyone have any thoughts on how I might get gdsHelpers to export an .oasis file so that I can compare filesize? Thanks so much for your time, and also thank you for developing such an amazing, open-source program!

Best,

-matt

![codeSnippet](https://user-images.githubusercontent.com/97685179/149410925-d7320282-00d8-4c3f-b0c6-4218dd7890ec.PNG)
![exportError](https://user-images.githubusercontent.com/97685179/149410939-e6e6d423-2422-42af-a582-3605558b2581.PNG)

**Comment by HelgeGehring** [comment]
Hey :)
Nice to hear that you like the package, thanks! :)

The documentation is at that point a bit wrong, for oasis-files the file-ending is expected to be ".oas" - I'll fix that :)
I didn't test too much about file-sizes, but I think .oas-files typically had half of the .gds-filesize.

And about the error: For exporting to dxf you need to install the package ezdxf.

Best,
Helge
